src/pathways/dto/create-pathway.dto.ts

```
@@ -0,0 +1,75 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import {
+  IsString,
+  IsNotEmpty,
+  IsBoolean,
+  IsNumber,
+  IsOptional,
+  MaxLength,
+  Min,
+  ValidateIf,
+} from 'class-validator';
```

ESLint

[error] Replace `'class-validator'` with `"class-validator"`

(prettier/prettier)